x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/time_scaling.tsx

```diff
@@ -117,9 +123,20 @@ export function TimeScaling({
     <EuiFormRow
       display="columnCompressed"
       fullWidth
-      label={i18n.translate('xpack.lens.indexPattern.timeScale.label', {
-        defaultMessage: 'Normalize by unit',
-      })}
+      label={
+        <EuiToolTip
+          content={i18n.translate('xpack.lens.indexPattern.timeScale.tooltip', {
+            defaultMessage:
+              'Normalized values are displayed as a rate, calculated from the date histogram interval. Values can scale down from a larger interval or scale up from a smaller interval. Partial intervals are handled correctly.',
+          })}
+        >
+          <span>
+            {i18n.translate('xpack.lens.indexPattern.timeScale.label', {
+              defaultMessage: 'Normalize by time unit',
+            })}
```

Comment:
Could you add an `EuiIcon` to this to indicate that there is a tooltip? I think the question mark in circle icon is standard here.